ext/extconf.rb

```diff
@@ -1,8 +1,15 @@
 require 'rubygems'
 require 'mkmf'
 
-$CFLAGS = " -fsigned-char -O3 -ggdb3 -mtune=native "
```

Comment:
manually setting `$CFLAGS` and `$CXXFLAGS` global variables is bad practice

Use `append_cflags` and `append_cppflags` as provided by mkmf.